openapi.yaml

```diff
@@ -0,0 +1,2094 @@
+        - dummy
+
+    WeightRule:
+      type: string
+      enum:
+        - downstream
+        - upstream
+        - absolute
+
+  # Reusable path, query, header and cookie parameters
+  parameters:
+    # Pagination parameters
+    PageOffset:
+      in: query
+      name: offset
+      required: false
+      schema:
+        type: integer
+        minimum: 0
+      description: The number of items to skip before starting to collect the result set.
+
+    PageLimit:
+      in: query
+      name: limit
+      required: false
+      schema:
+        type: integer
+        minimum: 1
+        default: 100
+      description: The numbers of items to return.
+
+    # Database entity fields
+    ConnectionID:
+      in: path
+      name: connection_id
+      schema:
+        type: integer
+      required: true
+      description: The connection ID
+
+    DAGID:
+      in: path
+      name: dag_id
+      schema:
+        type: integer
+      required: true
+      description: The DAG ID
+
+    DAGRunID:
+      in: path
+      name: dag_run_id
+      schema:
+        type: integer
+      required: true
+      description: The DAG Run ID
+
+    TaskID:
+      in: path
+      name: dag_id
+      schema:
+        type: integer
+      required: true
+      description: The Task ID
+
+    TaskTryNumber:
+      in: path
+      name: task_try_number
+      schema:
+        type: integer
+      required: true
+      description: The Task Try Number
+
+    EventLogID:
+      in: path
+      name: event_log_id
+      schema:
+        type: integer
+      required: true
+      description: The Event Log ID
+
+    ImportErrorID:
+      in: path
+      name: import_error_id
+      schema:
+        type: integer
+      required: true
+      description: The Event Log ID
+
+    PoolID:
+      in: path
+      name: pool_id
+      schema:
+        type: integer
+      required: true
+      description: The Pool ID
+
+    TaskRescheduleID:
+      in: path
+      name: task_fail_id
+      schema:
+        type: integer
+      required: true
+      description: The Task Fail ID
+
+    VariableID:
+      in: query
+      name: variable_id
+      schema:
+        type: string
+      required: true
+      description: The Variable ID
+
+    VariableKey:
+      in: query
+      name: variable_key
+      schema:
+        type: string
+      required: true
+      description: The Variable Key
+
+    ExecutionDate:
+      in: path
+      name: execution_date
+      schema:
+        type: string
+        format: 'date-time'
+      required: true
+      description: >
+        The date-time notation as defined by
+        [RFC 3339, section 5.6](https://tools.ietf.org/html/rfc3339#section-5.6), for example,
+        `2017-07-21T17:32:28Z`
+
+    # Logs
+    NoChunking:
+      in: query
+      name: full_content
+      schema:
+        type: boolean
+      required: false
+      description: A full reply will be returned. By default, only the first fragment will be returned.
```

Comment:
```suggestion
      description: A full reply will be returned. By default, only the first fragment will be returned
```

Reply:
In the descriptions, the dot at the end of the sentence makes sense.